src/applications/vaos/components/layout/CCLayout.jsx

```diff
@@ -57,12 +57,14 @@ export default function CCLayout({ data: appointment }) {
               </div>
             )}
         </When>
-        <What>{typeOfCareName || 'Type of care not noted'}</What>
+        <What>{typeOfCareName}</What>
         <Section heading="Provider">
```

Comment:
Hey @Cieramaddox I see that in the figma that all other appointments uses the "Who" heading to display the provider information with CC being the only type that uses the "Provider" heading. Just wanted to double check if this is intentional.